setup.py

```diff
@@ -51,7 +51,8 @@
         'ftw.profilehook',
         'Pillow',
         'ftw.colorbox',
-        'plone.api'
+        'plone.api',
+        'plone.batching>=1.0.4'
```

Comment:
@shylux basically everything OK. Jus one last question, why is it necessary to upgrade plone.batching to 1.0.4?